syntax = "proto3";

package pypkg;

service Ai {
    rpc GetSuggest (SuggestRequest) returns (SuggestResponse);
    rpc ClearHistory (ClearHistoryRequest) returns (ClearHistoryResponse);
}

message SuggestRequest {
    int64 uid = 1;
    string suggest = 2;
}

message SuggestResponse {
    bool ok = 1;
    string request = 2;
}

message ClearHistoryRequest {
    int64 uid = 1;
}
message ClearHistoryResponse {
    bool ok = 1;
}